### PR TITLE
12-29-2020 equity dashboard chart data update

### PR DESCRIPTION
**Pull request by wpserviceuser**
Equity dashboard stats updates in this PR may be reviewed on staging: https://staging.covid19.ca.gov/equity/

After reviewing, if all looks well, approve and merge this Pull Request.

If there are issues with the data:

- Note concerns or issues here by commenting on this PR

- Work with Triston directly to resolve data issues

- Alert the COVID19 site team in Slack (in the Equity page channel)

### CODE REVIEW

File: data/reviewed/equitydash/equitytopboxdatav2.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+      "STATE_CASE_RATE_PER_100K": 5338.91564139394,
+      "CASE_RATE_PER_100K": 5524.70538790969,
       "POPULATION": 7290434,
-      "CASES_7DAYAVG_7DAYSAGO": 365392,
-      "RATE_DIFF_30_DAYS": 1302.70708163602
+      "CASES_7DAYAVG_7DAYSAGO": 402775,
+      "RATE_DIFF_30_DAYS": 1605.17192803611
     }
   ],
   "Demographics": [
```
Comment (coreycaitlin) [inline data/reviewed/equitydash/equitytopboxdatav2.json:12]: Of the numbers that should have changed in this file:

- ✅ Case rate for all ethnicities/income brackets
- ✅ Case rate for low-income communities
- ❌ Death rate for Latino people
- ❌ Death rate for all ethnicities
- ❌ Case rate for NHPI
- ❌ Death rate for Black people

**Comment by coreycaitlin** [comment]
We have the correct number of files changed, several of the figures in the top boxes files didn't get updated. @TristonMosbacher and @jbum can you help sort out what happened here?

**Comment by TristonMosbacher** [comment]
Thanks for checking those, @coreycaitlin !  Taking a look now.

**Comment by coreycaitlin** [comment]
Looks like we have all the data now! Over to you @sindhuravuri and @vargoCDPH 🎉

**Comment by sindhuravuri** [comment]
Hi everyone! Before I post up my full review, I noticed something right away and wanted to make sure I got your feedback on it. I think the math for Blue Box 1 (Latino death rate) might be off - it is currently 23% but I suspect it should be around 16% according to the numbers (68.6 Latino vs 59.3 all ethnicities). @coreycaitlin @TristonMosbacher @vargoCDPH

**Comment by TristonMosbacher** [comment]
Just checked this too, and you're definitely right @sindhuravuri .  Looks like that percentage is from the previous week (last week was 68.6 Latino, 55.7 All Eths.).  Perhaps it didn't re-calc after the latest update?

**Comment by TristonMosbacher** [comment]
I think it's still pulling in the previous week's death rate for Latinos, actually.  Checking in Snowflake, I'm fairly certain that rate should be 72.8, which once again results in 23% higher than all ethnicities (59.3) for this week as well.  Unsure why that rate in particular remains the same while all others have changed.

File: data/reviewed/equitydash/equitytopboxdatav2.json [hunk 2]
```diff
@@ -12,66 +12,66 @@
...
-      "DEATHS": 10721,
+      "CASES": 874811,
+      "DEATHS": 11380,
       "POPULATION": 15624857,
-      "REPORT_DATE": "2020-12-21",
-      "CASE_RATE": 4969.6,
-      "DEATH_RATE": 68.6
+      "REPORT_DATE": "2020-12-28",
+      "CASE_RATE": 5598.8,
+      "DEATH_RATE": 72.8
```
Comment (coreycaitlin) [inline data/reviewed/equitydash/equitytopboxdatav2.json:47]: It looks like this number did update! Weird that it's not showing in staging.

**Comment by coreycaitlin** [comment]
Paging @jbum (and possibly @aaronhans? I know you had looked at what's up with the top boxes getting out of sync on staging) to sort this out.

**Comment by aaronhans** [comment]
Yeah, something is wrong in staging. When I run this locally I see the expected numbers which for the first box should be:
Screen Shot 2020-12-29 at 6 37 51 PM

Reviewing what is wrong with staging code

**Comment by sindhuravuri** [comment]
Hi again everyone — thank you very much for your help and explanation with the staging issue for Latino death rate. 

Below are my remaining notes for today’s data review — **no red flags on my end! Values seem reasonable and align with our previous data.** I saw only a few categories that I will continue to monitor for stabilization, but am not concerned about (Multi-Race deaths/100K rate, White vs. Latino testing/100K, and data completeness for gender identity). 

Thank you!

1. **Blue Box Updates:**
_Summary statement: All values appear reasonable and consistent with previous data; only discrepancy are values in Blue Box 1, which we are looking into._ 

2. **Death Rate/100K by R&E:**
_Summary: We see increases in all death/100K rates; all values and increasing trends are reasonable/expected, except the rate of increase for Multi-Race — this rate is unprecedented (Asian American & Multi-Race rates ranking switched); need to monitor Multi-Race closely to see how trend oscillates or stabilizes in coming week. Deaths relative to percentage of population in CA did not change significantly (even unknown) — aligns with expectations. AI/AN data now fully visible!_  

3. **Case Rate/100K by R&E:**
_Summary statement: We see reasonable increases in all case/100K rates (increasing trends align with expectations); ranking of Black vs. AI/AN & Multi-Race vs. Asian American switches; no significant changes in cases relative to percentage of population in CA (updates align with case/100K ranking changes and expectations)_

4. **Testing Rate/100K by R&E**
_Summary: Reasonable increases in all testing/100K rates; Latino testing rate increases significantly, changing ranking of Latino, Black and White categories — this is expected as Black testing rate/100K shows larger rate of increase than previous week, while White testing rate/100K maintains the same rate of increase as previous week. Will continue to monitor this. Testing relative to percentage of population in CA shows no significant changes, all numbers are stable and reflect changes in White, Black, Latino rankings_

5. **Mortality rate/100K < Case rate/100K for all categories (as expected)**

6. **Community case rate/100K by median annual household income bracket:**
_Summary: Cases/100K increased reasonably; 0-40K remains the highest and statewide case rate increased — all values align with expectations_

**7. HE Metric: successfully extended — extension aligns with 7-day lag; reasonable increase in health equity quartile positivity and statewide positivity** 

**8. Data completeness (DC): Deaths are the most complete for race/ethnicity and gender identity, not for sexual orientation — all numbers are reasonable and align with data trends from previous week; only unexpected trend is decrease in GI cases (previously was stable) — will monitor for stabilization in coming week**

**Comment by vargoCDPH** [comment]
> Yeah, something is wrong in staging. When I run this locally I see the expected numbers which for the first box should be:
> Screen Shot 2020-12-29 at 6 37 51 PM
> 
> Reviewing what is wrong with staging code

it appears this is fixed in staging now! thanks.

**Pull request merged**